src/controllers/playback/video/index.js

```diff
@@ -262,6 +262,26 @@ export default function (view) {
 
         LibraryMenu.setTitle(title);
 
+        // Make the page title clickable to navigate to the item detail page
+        const pageTitleElement = document.querySelector('.pageTitle');
```

Comment:
Better to query from `view` rather than `document` (to narrow the search).

Reply:
Oh, my bad, it belongs to the library menu.

As an option, we could add `url` argument to `LibraryMenu.setTitle` and add/remove click handling or add/change the link url there. Or maybe add another function for title "link", but `setTitle` should reset it anyway. I'm for `setTitle(title, url = '')`, but let's see what others say.